src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--0 > * {
+  background-image: url(/images/star.svg);
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:last-child {
+  background-image: url(/images/star.svg);
+}
```

Comment:
let's add this to `stars__star` selector, for all of the stars. And after that reassing it for those that need it